main.py

```diff
@@ -128,42 +113,42 @@ def main():
                         .output(output_path, vcodec="libx264", pix_fmt="yuv420p", acodec="aac", crf="23")
                         .run()
                     )
-        else:
-            print("ERROR: Path not valid.")
 
 
-def display_aspect_ratio(string):
-    # Function to get the DAR from a given aspect ratio string.
+def display_aspect_ratio(ratio_string):
+    # Get the numeric ratio from the aspect ratio string, e.g. "4:3" → 1.333(3)
 
-    vals = [int(val) for val in string.split(":")]
-    return vals[0] / vals[1]
+    numbers = ratio_string.split(':')
+    width = int(numbers[0])
+    height = int(numbers[1])
 
+    return width / height
```

Comment:
Not really a necessary change, but for me easier to read than the list comprehension - what do you think?